devtools/platform-descriptor-json/src/main/resources/templates/README.gradle.ftl

```diff
@@ -0,0 +1,38 @@
+# ${project_artifactId} project
+
+This project uses the Supersonic Subatomic Java Framework named Quarkus.
+
+If you want to learn more about this framework, please check this website: https://quarkus.io
+
+
```

Comment:
It's minor but could we remove the extra new lines in both files?

When we have 2 or 3, let's make it one. Better to have the file compact for when you want to read it.